pkg/skaffold/deploy/helm_test.go

```diff
@@ -272,6 +272,7 @@ var testTwoReleases = latest.HelmDeploy{
 	}},
 }
 
+var crateNamespaceFlag = true
```

Comment:
```suggestion
var createNamespaceFlag = true
```

Reply:
@gsquared94 thanks for catching this. fixed